scene/resources/canvas_item_material.h

```diff
@@ -59,6 +59,7 @@ class CanvasItemMaterial : public Material {
 			uint32_t light_mode : 4;
 			uint32_t particles_animation : 1;
 			uint32_t invalid_key : 1;
+			bool particles_anim_resize;
```

Comment:
Shouldn't this use `uint32_t particles_anim_resize : 1;` for consistent with the other elements above?